app/assets/stylesheets/views/profile.scss

```diff
@@ -142,3 +142,29 @@
     );
   }
 }
+
+.profile-comment-card {
+  border-radius: 0;
+
+  @media screen and (min-width: $breakpoint-s) {
+    border-radius: var(--radius);
+  }
+}
```

Comment:
@ludwiczakpawel I'm wondering if we might consider having a variation of `crayons-card` with these styles because I find I re-use sometimes. What do you think?

Reply:
Yea, I've actually [introduced something like this couple days ago](https://github.com/forem/forem/blob/master/app/assets/stylesheets/components/cards.scss#L21-L34), but lets keep it as you have it here for now, since my solution may need some tweaks :D